protocol/0071-STAK-erc20_governance_token_staking.md

```diff
@@ -90,4 +90,21 @@ Other functionality:
 * Attempts to redeem vested tokens will fail if there are not sufficient tokens held on behalf of the sender address that are not staked. Sender must first unstake tokens before they can be redeemed.
 * This functionality does not interact in any way with the staking bridge contract. They are effectively completely separate staking mechanisms, so to unstake all an address's tokens when some are staked on each contract will require calls to both contracts.
 
+# Acceptance Criteria
+### Staking Bridge Smart Contract 
+* Staking Bridge accepts and locks deposited VEGA tokens and emits `Stake_Deposited` event
+* Staking Bridge allows only stakers to remove their staked tokens and emits `Stake_Removed` event
+* Staking Bridge allows users with staked balance to transfer ownership of stake to new ethereum address that only the new address can remove
+* Staking Bridge prohibits withdrawal of VEGA while that VEGA is staked
+
+### Vesting Smart Contract 
+* Vesting Contract locks vesting VEGA tokens and emits `Stake_Deposited` event
+* Vesting Contract unlocks vesting VEGA tokens and emits `Stake_Deposited` event
+* Vesting Contract prohibits withdrawal of VEGA while that VEGA is staked
+
+### Event Queue
```

Comment:
Until now we were on Ethereum side but the Event Queue is vega chain side, correct?

Reply:
correct, but it makes sense that the Vega side would have to respond before this is all accepted right?

Reply:
Yes; I was just hoping to see a line to make it clear to people who are less familiar with this.